styles/img-slider.css

```diff
@@ -0,0 +1,12 @@
+.img-slider {
+  margin: 4rem 1rem;
+  height: 350px;
```

Comment:
use rem here as well

Reply:
this is just to some height, in future it will be removed by slider image.